sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb

```diff
@@ -157,7 +157,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        instance_type="ml.m5.xlarge",
</pre><p>Why do these all need to switch to this different instance type?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>These are the mis-usages I mentioned in the other comment.</p><p>The reason is <code>sagemaker.image_uris.retrieve</code>  is retrieving <code>image_uri</code> in PythonSDK but the <code>instance_type</code> assigned was a Parameter placeholder which can only be parsed during Pipeline backend.  In other words, the Parameter is not working at all in this case and we have received several customer complains on this.</p><p>We have a SDK change (not yet release) to prevent such behavior but to avoid the change break notebook build, we correct this usage at this moment.  </p>